modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/translation_manager/TranslationFilter.tsx

```diff
@@ -122,6 +122,11 @@ export default function TranslationFilter({
 		setSelectedKey(option as string);
 	};
 
+	useEffect(() => {
```

Comment:
There is another requirement that is missing I think, this one "If the user Save as Draft, the filter should keep its value", currently, if we save as draft, the filter is set to "All fields"

Reply:
This happens because we actually re-render the whole jsp when the user clicks save as draft.

The only way we could avoid that if we would keep the filter state in browser memory or in the request.I open a task for it and reach out to Ruth